disperser/apiserver/disperse_blob_v2.go

```diff
@@ -28,15 +46,15 @@ func (s *DispersalServerV2) DisperseBlob(ctx context.Context, req *pb.DisperseBl
 	// Validate the request
 	onchainState := s.onchainState.Load()
 	if onchainState == nil {
-		return nil, api.NewErrorInternal("onchain state is nil")
+		return nil, status.New(codes.Internal, "onchain state is not available")
```

Comment:
Building statuses this way bypasses the utility methods introduced by @samlaf. If we want to build statuses instead of errors, should we have equivalent utilities for statuses?